src/ape/api/providers.py

```diff
@@ -78,6 +71,15 @@ def validate_hexbytes(cls, value):
             raise ValueError(f"Hash `{value}` is not a valid Hexbyte.")
         return value
 
+    @validator("transaction_ids", each_item=True, pre=True)
+    def validate_transactions(cls, value):
+        return HexBytes(value)
```

Comment:
Just in case, I guess we should check that the value isn't already a `HexBytes` value?

```suggestion
    @validator("transaction_ids", each_item=True, pre=True)
    def validate_transactions(cls, value):
        if isinstance(value, HexBytes):
            return value
        return HexBytes(value)
```